src/lib.rs

```diff
@@ -60,9 +60,9 @@ use typeable::Typeable;
 /// `.downcast` or the `match_error!` macro. This type can be used to meet
 /// the needs of consumers like `Verifier`, but should not be used in cases
 /// where enums or generics are better suited.
-pub trait Error: Show + Send + Typeable + StdError { }
+pub trait Error: Debug + Send + Typeable + StdError { }
 
-impl<S: StdError + Show + Send + Typeable> Error for S { }
+impl<S: StdError + Debug + Send + Typeable> Error for S { }
```

Comment:
I'm not convinced this Debug bound is actually needed anymore, given that StdError now inherits from Display, but we can keep it for convenience for now.